libs/constants.js

```diff
@@ -5,8 +5,11 @@ export const ACTIVE_SIGNATURE_PENDING = 'active:signature:pending';
 export const ACTIVE_SUBMITTED = 'active:submitted';
 export const ACTIVE_PROCESSING = 'active:processing';
 export const CLOSED = 'closed';
+
 export const NOT_STARTED_VIVA = 'notStarted:viva';
 export const ACTIVE_COMPLETION_REQUIRED_VIVA = 'active:completionRequired:viva';
+export const ACTIVE_COMPLETION_RANDOM_CHECK_REQUIRED_VIVA =
```

Comment:
Can we rename this to `ACTIVE_RANDOM_CHECK_REQUIRED_VIVA`? The current name feels misleading, as `COMPLETION` is saying we need to send in additional details.

Reply:
Will fix